canonical-web-and-design/jaas-dashboard
# Update dependencies

## Done

Update dependencies

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- CI should pass, the app should work.
